### PR TITLE
Add fixture for population projection

### PR DESCRIPTION
**Pull request by Catacola**
## Description of the change

Add two new fixture files.

- `server/core/demo_data/population_projection_timeseries.json`
  - Contains the timeseries data for population projections
  - Fields
    - "year": year for the data
    - "month": month for the data
    - "compartment": "INCARCERATION"|"SUPERVISION",
    - "legal_status": rider, termer, revocation, general, etc,
    - "simulation_group": "FEMALE"|"MALE",
    - "state_code": "US_DEMO",
    - "simulation_tag": "HISTORICAL"|id of model used for this projection,
    - "total_population": expected or recorded population at this point in time,
    - "total_population_min": lower bound for projection
    - "total_population_max": lower bound for projection
- `server/core/demo_data/population_projection_summaries.json`
  - Contains summary data for each view (releases, admissions, population, error bars)
  - Fields
    - "state_code": the state
    - "metric_period_months": number of months out for this summary
    - "simulation_date": when this simulation was run
    - "compartment": "INCARCERATION"|"SUPERVISION"
    - "legal_status": rider, termer, revocation, etc
    - "simulation_group": "MALE"|"FEMALE"
    - "simulation_tag": "HISTORICAL"|tag for model used for this projection
    - "admission_count": total intakes over period
    - "release_count": total releases over period
    - "total_population": population at the end of the period
    - "admission_count_(min|max)": upper and lower bounds for admission count projection. NULL for HISTORICAL
    - "release_count_(min|max)": upper and lower bounds for release count projection. NULL for HISTORICAL
    - "total_population_(min|max)": upper and lower bounds for population projection. NULL for HISTORICAL

~NOTE: There is a bug on master currently which makes the community goals view crash in test mode:~
Apparently I just needed to run `yarn install` again.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Closes https://github.com/Recidiviz/recidiviz-data/issues/6215 (I know. Wrong repo.)

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by daschi** [comment]
Hmm I'm not able to reproduce the bug, demo mode is loading fine for me on all the ND views.

File: server/core/demo_data/population_projection_summaries.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    "release_count_max": "1000",
+    "total_population_count_min": "12890",
+    "total_population_count_max": "13110",
+    "admission_percent_change": "3.8",
+    "release_percent_change": "8",
+    "population_percent_change": "-3.1"
+  },
+  {
+    "state_code": "US_DEMO",
+    "metric_period_months": "36",
```
Comment (daschi) [inline server/core/demo_data/population_projection_summaries.json:55]: Was it intentional to only have 6 and 36 months for these?

**Review by daschi — APPROVED** [review]

File: server/core/demo_data/population_projection_timeseries.json [hunk 2]
```diff
@@ -0,0 +1,13466 @@
...
+    "year": "2018",
+    "month": "1",
+    "compartment": "INCARCERATION",
+    "legal_status": "GENERAL",
+    "simulation_group": "FEMALE",
+    "state_code": "US_DEMO",
+    "simulation_tag": "HISTORICAL",
+    "total_population": "1009.5",
+    "total_population_min": "1009.5",
+    "total_population_max": "1009.5"
```
Comment (daschi) [inline server/core/demo_data/population_projection_timeseries.json:12]: will this be updated to include all the attributes?

File: server/core/demo_data/population_projection_summaries.json (hunk 1 above)
Comment (Catacola) [reply server/core/demo_data/population_projection_summaries.json:55]: Yup! Justine is still making the actual views which will back this. I just made up entries for 2 views so we had something to develop against.

File: server/core/demo_data/population_projection_timeseries.json (hunk 2 above)
Comment (Catacola) [reply server/core/demo_data/population_projection_timeseries.json:12]: At least for now, the plot will only show total population over time. Breakdown by admission and release will only be in the summary box.

**Pull request merged**